### ISSUE TITLE
512px by 512px input requirement unclear

### ISSUE THREAD

**Issue by deeppppp** [deepinstruction/deepnude_official#2]
If the input image is not 512px by 512px the algorithm will fail.
Please allow for dynamic width / height properties for the input image.

When there was a GUI available, it would crop the input to the correct size. Without a GUI this input requirement is obscured and will likely trip up many users.

**Comment by rinukkusu** [comment]
I've already tried to use the `.shape` field of the cv images instead of the hardcoded 512px - this correctly outputs an image of the same size as the `input.png`, but the results are rather poor.

**Comment by niko247** [comment]
yeah I think this is the first thing we should do, but I'm not sure if this is possible maybe the encrypted file *.lib have hardcoded data only for 512x512 files so it will be hard for good result
